### PR TITLE
Add Auth Swift Framework Test.

### PR DESCRIPTION
**Pull request by granluo**

### CODE REVIEW

**Comment by granluo** [comment]
Test is done in bc507d6

File: .github/workflows/auth.yml [hunk 1]
```diff
@@ -76,6 +77,43 @@ jobs:
       # Only build the unit tests on Catalyst
       run: scripts/test_catalyst.sh FirebaseAuth build FirebaseAuth-Unit-unit
 
+  quickstart_framework:
+    env:
```
Comment (paulb777) [inline .github/workflows/auth.yml:81]: Missing

`    # Don't run on private repo.
    if: github.event_name == 'schedule' && github.repository != 'FirebasePrivate/firebase-ios-sdk'`

Also looks like a problem in Storage, DynamicLinks and Database. The scheduled jobs on https://github.com/FirebasePrivate/firebase-ios-sdk/actions should be over in seconds.

Comment (granluo) [reply .github/workflows/auth.yml:81]: Will add that in the pr.
Looks like I have no access to FirebasePrivate. I cannot get into the link.

Comment (granluo) [reply .github/workflows/auth.yml:81]: Will update those three in the pr as well.

Comment (granluo) [reply .github/workflows/auth.yml:81]: Just made another [pr](https://github.com/firebase/firebase-ios-sdk/pull/5968) 😅  to resolve this.

**Pull request merged**